imagepipeline/src/main/java/com/facebook/imagepipeline/core/ImagePipeline.java

```diff
@@ -423,7 +423,24 @@ public boolean isInBitmapMemoryCache(final ImageRequest imageRequest) {
    * @return true if the image was found in the disk cache, false otherwise.
    */
   public boolean isInDiskCacheSync(final Uri uri) {
-    return isInDiskCacheSync(ImageRequest.fromUri(uri));
+    return isInDiskCacheSync(uri, ImageRequest.CacheChoice.DEFAULT);
```

Comment:
Can we check both caches here? `isInDiskCacheSync(uri, CacheChoice.SMALL) || isInDiskCacheSync(uri, CacheChoice.DEFAULT)`

As we have the in-memory index for the disk cache now, this should be much faster anyway